CardanoSharp.Wallet/Models/Addresses/Address.cs

```diff
@@ -0,0 +1,131 @@
+﻿using CardanoSharp.Wallet.Encoding;
+using CardanoSharp.Wallet.Enums;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace CardanoSharp.Wallet.Models.Addresses
+{
+    public class Address : IEqualityComparer<Address>, IEquatable<Address>, IEquatable<string>, IEquatable<byte[]>
+    {
+        private byte[] _bytes;
+        private string _address;
```

Comment:
There have been times when having access to both of would have been very useful.

Reply:
ToString() gives you the Address
GetBytes() gives you the Bytes

the rationale for using methods instead of properties is that properties don't support formatting or endianness....  maybe that is a feature for the future...

Reply:
One thing that we need to do is make a string encoder so people can query by hash. https://github.com/CardanoSharp/cardanosharp-demo/blob/CreateTransactionExplorer/src/Infastructure/Persistence/CardanoEFCore/Queries.cs If you look here at line 70, I did a raw SQL query from the passed in hash. The Postgres hash type is a 32 byte array I believe, and when trying to convert a string to the proper Postgres type it was a pain that I couldn't get. So I think it would be super useful to have an extenstion method that could turn strings into that postgres types.

Reply:
Ill dive into how the tx id is created. I know the cardano cli can tell you want it will be before you even submit so im sure we can figure it out. Im sure its something along the lines of byte[] -> blake hash